pages/templates/pages/microsite_page.html

```diff
@@ -1,12 +1,210 @@
 {% extends "pages/base_page.html" %}
 
-{% load wagtailcore_tags %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block meta %}
+  {{ block.super }}
+  <style>
+    #custom_header {
+      {% if page.custom_header_background_color %}
+        background-color: #{{ page.custom_header_background_color }};
+      {% else %}
+        background: transparent;
+      {% endif %}
+
+      {% if page.show_accent_border %}
+        border-top: solid 8px #{{page.accent_border_color}};
+      {% endif %}
+    }
+
+    #primary-cta {
+      {% if page.button_background_color %}
+        background-color: #{{page.button_background_color}};
+        border-color: #{{page.button_background_color}};
+      {% endif %}
+
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}};
+      {% endif %}
+    }
+
+    .microsite-content a#primary-cta {
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}};
+      {% endif %}
+    }
+
+    /* TODO: Add hover color to model or dynamically generate */
+    #primary-cta:hover {
+      opacity: 0.9;
+      transition: 0.25s;
+    }
+
+    #primary_content {
+      background-size: cover;
+      {% if page.primary_content_background_color %}
+        background-color: #{{ page.primary_content_background_color }};
+      {% endif %}
+
+      {% if page.primary_content_text_color %}
+        color: #{{ page.primary_content_text_color }};
+      {% endif %}
+
+      {% if page.primary_content_background_image %}
+        {% image page.primary_content_background_image original as primary_img %}
```

Comment:
Why need this tag? Image url is not in model already?

Reply:
It returns an object and there is not a way to access URL without this afaik. This is recommended way to handle images in Wagtail http://docs.wagtail.io/en/v1.9/topics/images.html